### PR TITLE
Topic 4 - first version

### PR DESCRIPTION
**Pull request by DiegoJStroppiana**
Here is all exercises of Topic 4.

### CODE REVIEW

File: Exercise7-JDBC Connection/src/data/ConnectionDB.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      public ConnectionDB() throws SQLException{
+      try {
+		Class.forName("com.mysql.jdbc.Driver");
+	    connect = DriverManager.getConnection("jdbc:mysql://localhost/high_school2","root","Globant2016");
+	    
+	    
+	  } catch (ClassNotFoundException e) {
+		e.printStackTrace();
+	  }
+      statement = connect.createStatement();
```
Comment (gastonabastoG) [inline Exercise7-JDBC Connection/src/data/ConnectionDB.java]: Why do you put this in the constuctor of the class?

File: Exercise7-JDBC Connection/src/main/MainClass.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package main;
+
+import java.sql.SQLException;
+
+import data.ConnectionDB;
+
+public class MainClass {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
```
Comment (gastonabastoG) [inline Exercise7-JDBC Connection/src/main/MainClass.java]: Remember to delete TODO comments

File: Exercise7-JDBC Connection/src/main/MainClass.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	    try {
+			new ConnectionDB();
+		} catch (SQLException e) {
+			// TODO Auto-generated catch block
```
Comment (gastonabastoG) [inline Exercise7-JDBC Connection/src/main/MainClass.java]: Remember to delete TODO comments

**Review by gastonabastoG — COMMENTED** [review]
Revise my comments

File: Exercise7-JDBC Connection/src/data/ConnectionDB.java (hunk 1 above)
Comment (DiegoJStroppiana) [reply Exercise7-JDBC Connection/src/data/ConnectionDB.java]: I've changed it and created a new public method in the class to consult teachers

**Comment by DiegoJStroppiana** [comment]
I've also created bootCampTopic4 folder and put all files from this topic into this

**Pull request merged**